%% Author: Edgar Tejada
%% This is 2018-10-17-Annotated-Bibliography.

\documentclass [12pt]{article}
\title{Circadian Proteins and the Metabolic Cycles of Three Kingdoms \\\medskip A Literature Review}
\author{Edgar Tejada ([redacted])\\University of California, Merced}
\usepackage[utf8]{inputenc}
\usepackage[backend=biber, bibencoding=utf8, style=science, citestyle=numeric]{biblatex}
\usepackage[greek,english]{babel} %Needed for greek letters.
\usepackage[pdfborder={0 0 0.5 [3 3]}, pdftitle={Circadian Proteins and the Metabolic Cycles of Three Kingdoms}]{hyperref} %Needed for website
\usepackage{csquotes}
\addbibresource{ActualBib.bib}
%To append abstracts to *any* citation style, we will use stack exchange users tmgriffiths and moewe: 
%https://tex.stackexchange.com/a/278419
\DeclareFieldFormat{abstract}{\par\small\textbf{Abstract}: #1}
\renewbibmacro*{finentry}{%
	\iffieldundef{abstract}
	{%
		\finentry
	}
	{%
		\finentrypunct\printfield{abstract}\finentry
	}
}
\usepackage{fancyhdr} %Update header
	\lhead{Tejada, \thepage}
	\pagestyle{fancy}
	\renewcommand{\footrulewidth}{0pt}


\begin{document}
\maketitle



\begin{abstract}
	\indent For three of the major kingdoms of life (Bacteria, Animal, and Plant) energy intake and metabolism take a significant investment of time and resources, and yet these processes can only be done at times when energy is readily available. To improve fitness, all three kingdoms have, in parallel, developed their own protein sets that oscillate and regulate costly metabolic functions with exposure to blue light. A growing field of research on these various circadian proteins show that they all share functions and relation to cell divisions and efficacy,  but their mechanisms and and interconnection with general cell behavior are still not well known. Thus, this overarching review of current literature for the behavior and modeling of this class of proteins in four different organisms will showcase the parallel knowledge base for the three kingdoms: (i) That researchers at various lab have made great strides in correlating an organism's unique circadian proteins with lower metabolism, (ii) That researchers still do not have good predictive models for gene to protein expression that can account for inter-protein interactions, and (iii) that any medical applications are still out of reach.
\end{abstract}
\newpage
\section{Introduction}
\indent For three of the major kingdoms of life (Bacteria, Animal, and Plant) energy intake and metabolism take a significant investment of time and resources, yet still these processes can only be done at times when energy is readily available. For most organisms in the three kingdoms, the time with greatest access to energy sources is during the day, a time frame that also correlates with greater sunlight. To improve fitness, all three kingdoms have, in parallel, developed their own protein sets that oscillate and regulate costly metabolic functions with exposure to blue light, the most abundant visible light component of sunlight~\autocite{Cohen2015}. \par
Questions remain as to how these mechanisms work, how these proteins effect general cellular behavior, and how alterations to the exposure to blue light alters these proteins and their affects within cells. Several researchers strive to study and generate robust predictive models for these cycles, and to do so they compare transgenic organisms that lack correct versions of these genes to wild variants~\autocite{Kepsutlu2014}. This review will display the current state of affairs for each kingdom, focusing first on studies in simpler cyanobacteria~\autocite{Tseng2014}, then on similar a study in mice~\autocite{Yamajuku2011}, followed by a study on a mutant flowering plant vs its wild counterpart~\autocite{Kim2008}, and ending on the current, early stages of studies with human tissue~\autocite{Keller2009}. In each case, this review will showcase: \par (i) that researchers at various labs have made great strides in correlating an organism's unique circadian proteins with lower metabolism, \par (ii) that researchers still do not have robust predictive models for gene to protein to behavioural expression that can account for inter-protein interactions, and \par (iii) that any biomedical applications are still out of reach.
\newpage
\section{The Simple Nightlife of Cyanobacteria}
	\indent Cyanobacteria hold a unique place in bacteria in that they regularly photosynthesize to produce oxygen, still regularly consume any excess food sources like traditional heterotrophs, have several lineages whose genomes are completely mapped, and are the only known lineage of bacteria that regulate their metabolic cycles relative to blue light~\autocite{Tseng2014}. As this class of unusual bacteria are so well studied, several branches are ideal candidates for studying the gene to protein to metabolic regulation pathways. These {`}simple' bacteria are common models for various experiments in the literature, but it is here where their nature as opportunists shine. Recent experiments focused on potential circadian proteins that may regulate the consumption of readily available sugar relative to blue light availability. \par
	\indent One set of such researchers, Osanai and group from the RIKEN Center for Sustainable Resource Science, compared a control cyanobacteria lineage against a transgenic line whose HOX80 gene was over-expressed, that is enhanced beyond the control group's, to test the effects of the gene on glucose uptake both at night and during the day.~\autocite{Osanai2014} Osanai and colleagues found that over expression of this gene both led to the exaggerated bacteria being unable to intake sugar at levels even during day-like blue light. The experimental groups sugar consumption was only comparable only to the control's night time consumption, in that was extremely low. The lower reaction level  of sugar in both groups correlated with higher levels of SigE proteins in the cells, but only the the HOX80 exaggerated subjects died after 4 days in total darkness. While this suggests SigE protein and the suspect gene are {`}metabolic dampeners' that tell the cell to be less active at night, the RIKEN lab did not observe any translation between gene HOX80 and the SigE, let a lone a mechanical model for how SigE might suppress the metabolizing of sugars.~\autocite{Osanai2014} \par
	\indent During roughly the same period of 2014, a comparative study lead by Tseng, of the UC San Diego Center for Chronobiology, ran a similar experiment on a lineage of cyanobacteria whose KaiB encoding genes were {`}knocked down', that is suppressed via replacing those genes with dummy genes that do nothing. The Chronobiology lab found that under similar tests of long exposure to darkness and blue light, both the control and experimental groups under different incubation periods still survived. In this case, the Tseng lab also used electron microscopy to detect and account for the inter-protein interactions in the cells~\autocite{Tseng2014}. In this case, Tseng and colleagues found a competitive relationship between KaiA and KaiC, two different but know circadian proteins, that fought for control of the cell's ATP intake in the absence of KaiB.  Indeed, the Tseng lab generated a stronger predictive model of KaiB acting as a more of a regulator between the two different oscillation cycles of its competing pair then as a true circadian protein in and of itself. \par
	\indent In both cases, the seemingly simple design and known genome of cyanobacteria betrays the hidden complexity. The circadian {`}rest' cycle is more than the sum of the parts of the proteins encoded by the genes, it is also the interactions between those proteins. For the Osanai lab, correlation but not causation was found for one gene to protein relation with energy uptake, while with the Tseng lab, a mechanical model disqualified one such protein, KaiB, from being a true circadian oscillator; instead the Tseng lab showed that yet unknown interactions lead to emergent protein to to protein regulatory behavior~\autocite{Tseng2014}.
\newpage
\section{E4BP4 Knock-down in Mice also Knocks-down Miceâ€™s Longevity}
	\indent Mice also make ideal organisms for various studies due to their short lifespans and well documented genome~\autocite{Shifman2006}, and just like cyanobacteria, mice are treated as {`}simplified'  models for all other mammals, including humans. With this in mind, contemporary research groups have sought to study and make predictive mice models for gene deficiencies in potential circadian proteins in the hopes of finding avenues to human treatments~\autocite{Cohen2015}. For example, the Astella Inc's Drug Discovery and Research Lab, lead by Yamajuku, published a scientific letter detailing their 2011 endeavors into studying the complexity of circadian regulation of mice tissue via both the suppression and over expression of target protein encoding genes. \par

	\indent The Yamajuku lab built off of and acknowledged the known framework that certain proteins, such as the CRY1 and CRY2 series, are part of the tissue to tissue hierarchy of time controlled metabolic regulation, with the nervous system being the most dominant system~\autocite{Yamajuku2011}.  Yamajuku and colleagues avoided this issue by taking cell samples from mice, removing the E4BP4 encoding gene, and growing the new modified tissue as liver fibroblasts, that is isolated cell growth in vitro. By doing this, the Astella lab avoided some of the inter tissue communication that complicated an already nebulous picture of what encoding proteins do what for the cells metabolism relative to the day cycle. After eliminating these confounding variables, this study found increased light exposure and knocking out E4BP4 actually resulted in longer periods of  high metabolic activity, and thus inversely much shorter life spans for the cell cultures~\autocite{Yamajuku2011}. This somewhat agrees with existing studies on in vivo (full, transgenic) mice who also had E4BP4 suppressed, as the engineered mice's lifespan were often reduced by more than half~\autocite{Cohen2015} ~\autocite{Yamajuku2011}. \par
	\indent Still, not all was good for the expressed goal of drug delivery for the Yamajuku lab, as uncontrolled inter-protein reactions and various unknown unknowns.  The lab expressed in their conclusion that even in just isolated tissue, {``}[O]ur results suggest that the period length of mPer2 oscillation is more sensitive to modulation of the amounts of the D-box regulators rather than the E-box and RRE regulators, raising the possibility that, among the three clock elements, D-box regulators play a [more]  prominent role in determining the period length in the cellular circadian clock''~\autocite{Yamajuku2011}. Their findings, while promising, came to the same conclusion as the groups who focused on cyanobacteria: that circadian rhythm, even for just liver cells, is a complex, emergent system that needs all players studied holistically before a predictive model can be obtained. Later studies that attempted this multi-view approach to study in vivo interactions in the same type of mice liver cells ultimately dealt with so much variance in the results of gene suppression ,lethargy, and rates of death in mice models that no strong conclusions could yet be made.~\autocite{Kepsutlu2014} It should be noted that for both bacterial and mice studies, changes in the period of circadian cycles always correlated with changes in the metabolic strength of the cell, but this will prove to not be universal.
\newpage
\section{The Odd Case of Flowering Plant Resilience}
	\indent Given the shared means of photosynthesis and a possible common lineage via chloroplasts, an adamant reader might expect that suppression of similar genes in plants as was done in cyanobacteria models would lead to similar experiments~\autocite{Cohen2015}. However, when Kim and colleagues, of Pohang University, studied the circadian control proteins in gene suppressed Arabidopsis,  a flowering plant, they found a relation that differed from experiments on bacterial cells and mammalian tissue. While earlier studies found inverse relations between metabolic activity and the robustness (longevity) of cell cultures in the two other kingdoms,~\autocite{Yamajuku2011}~\autocite{Osanai2014}, the Kim research group found no resultant change in the metabolic activity of the flowering plant's cells and knock down changes to FIO1, a known regulatory circadian protein in other organisms~\autocite{Kim2008}. To clarify, the Kim group noted that the period of activity changed, but rate and cell health did not change. \par
	\indent To make matters more ambigious, the altered plant lineage did grow to maturity faster, which might have indicated lower robustness at the cost of cells never resting, but the long run experiment both with and without blue light exposure showed that the knockdown variant had no significant change to cellular viability nor general leaf growth towards sunlight. This somehow suggests that the flowering plant's cells faces no cost for the faster rate of growth nor for running its cells at full energy consumption at night~\autocite{Kim2008}. The study from Pohang University acknowledges this discrepancy, and suggests that in this subject plant, FIO1 gene may encode for an inter-protein regulator, and not a true circadian oscillator as FIO1 gene would in other organisms. Still, this suggests a more fundamental lack of understanding into the mechanics of how these proteins supress metabolic activity in cells in general. This poses a significant barrier to groups hoping to use treat humans who have circadian deficiencies.
\newpage
\section{Human Tissue Trials and Conclusions}
	\indent Sleep, cell regeneration, and the general health of the immune system are interconnected in ways that are not yet completely understood~\autocite{Keller2009}, but this does not discourage research and development in circadian protein modeling with the goal of drug delivery in mind. A meta-review of cyanobacteria by Dr. Cohen, of UC San Diego, noted  in their ending remarks that the view of cellular clocks as purely linear functions was fading away in favor of a more {`}network' based view of protein on protein behavior leading to cellular behavior~\autocite{Cohen2015}. This of course, complicates any research group trying to fill in the gaps of potential links between sleep and metabolic deficiencies in humans and the known human genome. \par
	\indent One such study, lead by Keller, found that some 8\% of transcriptions in human macrophages, a type of immune cell, are likely regulated by a handful of known circadian proteins. Keller and group go further, to argue that the immune system may have its own autonomous cellular clocks, isolated from the hierarchy of the nervous system~\autocite{Keller2009}. While Keller's findings did show strong correlation between the immune system and the CRY1 and CRY2 protein series, these proteins are known to be versatile in their use outside of sleep and metabolic regulation in mammal cells~\autocite{Yamajuku2011}. Such in vitro, culture based tests for human cell samples are also obfuscated by yet unknown protein interactions, as would be found in living mammals. Keller's closing remarks calls for more research into studying each possible confounding variable for the immune system's cellular clocks, remarking {``}We cannot exclude, however, that systemic time cues such as glucocorticoids, melatonin, or adrenergic/noradrenergic hormones play a role in synchronization the LPS response pathway that link the macrophage-intrinsic shaping of an immune response in the intact organism''~\autocite{Keller2009}. Much like with the studies of mice, approximations of all of the  individuals in the circadian system cannot account for the behavior of the whole, dampening current researcher's prospects of using the contemporary body of literature for drug development. Simply put, the path to medical application of circadian protein based drugs requires far more robust and complete studies of the circadian networks and gene encoding than currently available.
\newpage
\section{Acknowledgments}
	\indent This work was supported by the University of California, Merced and the Kolligan Library staff, both of which provided database access and resources. \par
	\indent We thank the hardworking and helpful staff at University of California, Merced's Writing Center for reviewing this and other work. \par
	\indent We do recognize the conflict of interest caused by this work and simultaneously applying to Dr. LiWang's Chronobiology lab.
\newpage
\medskip %Median skip?
\nocite{*} %This should go before we print the bibliography
\printbibliography %This only works with biber as the biblatex backend. :/



\end{document}
%List of source names, for self reference:
%~\autocite{Cohen2015}
%~\autocite{Keller2009}
%~\autocite{Kepsutlu2014}
%~\autocite{Kim2008}
%~\autocite{Osanai2014}
%~\autocite{Tseng2014}
%~\autocite{Yamajuku2011}
%~\autocite{Shifman2006}
